app/views/teams/list.html.erb

```diff
@@ -13,6 +13,15 @@
 <% else %>
   <% modelType = 'CourseTeam' %>
 <% end %>
+
+<% if modelType == 'AssignmentTeam' %>
```

Comment:
This is not the write thing to do. 
On the teams/list page, if the current object is an assignment, you will see a section caller "inherit teams from course". But you will not see this section if the current object is a course.